### ISSUE TITLE
`config` subcommand should make apparent which values it's actually using

### ISSUE THREAD

**Issue by tamarrow** [dcos/dcos-cli#754]
Currently,  each config property has a corresponding environment variable, that if set, overrides the stored value of that config property:

For example, if a user executes:
`dcos config set core.dcos_url http://foo`
`export DCOS_URL=http://bar`

The value the CLI will use for "core.dcos_url" will be`http://bar`. However, this is not reflected in the `config` commands.

Currently:
`dcos config show` ->

```
core.dcos_url=http://foo
```

Proposal:
Differentiate between "stored" and "using".
`dcos config show` ->

```
core.dcos_url 
    Using: http://bar ($DCOS_URL)
    Stored: http://foo (~/.dcos/dcos.toml)
```

**Comment by jsancio** [comment]
Cool. I suspect that the common case will be that all of the values come from config file and do not get overwritten by the env variable. In that case I think we want to keep the output the same. If the value comes from the env then we can output:

```
core.dcos_url=http://bar  # overwritten by $DCOS_URL; config file value: http://foo
```

**Comment by tamarrow** [comment]
fixed by #891
